docker/unitTest.sh

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Please add
`#!/usr/bin/env bash`
and license header.

Reply:
Done